src/discord/commands/book.ts

```diff
@@ -0,0 +1,229 @@
+import { CommandInteraction, MessageEmbed, MessageActionRow, MessageSelectMenu, MessageButton, ButtonInteraction, SelectMenuInteraction, Message } from 'discord.js';
+import roomModel, { Room } from '../../models/room.model';
+import sectionModel, { Section } from '../../models/section.model';
+import Timeblock from '../../models/timeblock.model';
+import { DateTime } from 'luxon';
+import { Types } from 'mongoose';
+
+interface TimeblockInformation {
+    startsAt: Date;
+    endsAt: Date;
+    availableCapacity: number;
+}
+
+interface sectionInformation extends Section {
```

Comment:
```suggestion
interface SectionInformation extends Section {
```

Nitpick but better for consistency and readability